radicle-cli/examples/workflow/1-new-project.md

```diff
@@ -0,0 +1,30 @@
+
+To create your first radicle project, navigate to a git repository, and run
+the `init` command:
```

Comment:
I think it would be helpful to let a first time user know that `rad auth` is needed before doing their first `rad init`. 

It can be added in this file as it's `1-new-project`, or to be added in some place like `0-quick-start.md` ?

Reply:
> Thanks for doing this. Very useful +1 ! 

Appreciated :) but its all a part of the job :)

> I think it would be helpful to let a first time user know that rad auth is needed before doing their first rad init.

Good point

Reply:
Its difficult to include `rad auth` in the workflow, so its suggested instead with a link to the documentation.